cmd/root.go

```diff
@@ -50,6 +50,7 @@ func GetRootCmd(args []string) *cobra.Command {
 	rootCmd.AddCommand(versionCommand())
 	rootCmd.AddCommand(generateCommand())
 	rootCmd.AddCommand(topologyCommand())
+	rootCmd.AddCommand(dnsCommand())
```

Comment:
I am wondering about having this here. The thought that is in my head is that we should only show this command if the plugin is in the path, but that means none of the get plugin logic that you have done would be needed. 

Then the though of well how does the user get the plugin in the first place comes to mind. With that I wonder if it makes sense to package to the plugin with a release of the kuadrantctl. But again I am not sure what the best option would be.

Reply:
If we want to ensure the presence of the plugin binary, I think it makes sense that a release of kuadrantctl would also package the binaries for the plugins. This would mean a change in the release process for kuadrantctl.